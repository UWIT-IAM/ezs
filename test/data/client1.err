mycert cn=localhost
.. Connect, ezs error 1 SSL error
error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed
mycert cn=localhost
.. Connect, ezs error 1 SSL error
error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed
mycert cn=localhost
server_reader
saving session
stdin: Success
using recovered session
mycert cn=localhost
server_reader
saving session
stdin: Success
