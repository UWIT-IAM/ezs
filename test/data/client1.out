Connection established to 'localhost:98765'.
> send 26 bytes
> send 15 bytes
> Connection established to 'localhost:98765'.
> send 27 bytes
> send 5 bytes
> 